modal/_utils/grpc_utils.py

```diff
@@ -118,61 +113,22 @@ async def send_request(event: grpclib.events.SendRequest) -> None:
     return channel
 
 
-_Value = Union[str, bytes]
-_MetadataLike = Union[Mapping[str, _Value], Collection[Tuple[str, _Value]]]
-
-
-class UnaryUnaryWrapper(Generic[RequestType, ResponseType]):
```

Comment:
Moving these to the `modal.client` module since they are now tied to a client instance